### ISSUE TITLE
Only use https links in HTML

### ISSUE THREAD

**Issue by acka47** [hbz/lobid#401]
As switching to https everywhere with #352 wasn't a good idea, we should at least only use https links in the HTML (and maybe add redirects for `lobid.org`, `lobid.org/resources`, `lobid.org/gnd` and `lobid.org/organisations)` so that browsers at least have it https everywhere.

lobid-organisations already uses https in all internal links as far as I can see. I thinks it already works fine.

- [ ] On the [team page](https://lobid.org/team/) we should use https links to the member pages.
- [ ] lobid-resources uses http links in result pages and other internal links.
- [ ] While lobid-gnd already is using https links internally, it uses some http links to external vocabularies (Sachgruppen, Geschlecht and Ländercodes) and EntityFacts. I don't know whether we have to fix that.

Please open separate issues in the corresponding repos if needed.

**Comment by dr0i** [comment]
>  Also, on the team page (even with https) we should use https links to the member pages

yeah, but that means to have https enabled domains. E.g. mine hasn't. And now to just link to them using https result in a SEC_ERROR_EXPIRED_ISSUER_CERTIFICATE and in newer browser you cannot ignore that, meaning you can't lookup the page.

**Comment by dr0i** [comment]
The solution for the team-page, to have get a green secure lobid page,  would be in not embedding pictures taken from http-domains but e.g. to copy the pictures locally and serve them from there.

**Comment by acka47** [comment]
I was just talking about the links from the team page to the individual profiles. They are always http.

**Comment by dr0i** [comment]
@fsteeg I think the landing pages of `resources`, `organisations` and `gnd` are treated via their play app instances, yes? Then we shall open issues in the respected repos, like @acka47 said.

**Comment by acka47** [comment]
Currently, the "language" icon (🌎) and the "information" icon (🛈) are broken in both chrome and firefox when viewing the non-https version, see http://lobid.org/. This would be solved when using https everywhere for this page (and for the landing apges of the three services).

**Comment by acka47** [comment]
I think we should solve this issue with a redirect (https only) for the landing pages of lobid, team and the three services.

**Comment by dr0i** [comment]
Re "icons": these were fixed by setting the CORS header. @acka47 please ack.

**Comment by acka47** [comment]
> Re "icons": these were fixed by setting the CORS header. @acka47 please ack.

+1
